src/test/java/unicash/logic/parser/FindCommandParserTest.java

```diff
@@ -86,8 +90,111 @@ public void toStringMethod() {
                 new TransactionContainsAllKeywordsPredicate();
 
         String expected = new ToStringBuilder(new FindCommandParser())
-                .add("findPredicate", findPredicate).toString();
+                .add("findPredicate", findPredicate)
+                .toString();
+
         assertEquals(expected, findCommandParser.toString());
     }
 
+    @Test
+    public void parseMethod_inputContainsDateTimePrefix_throwsException() {
```

Comment:
Best to include name of exception thrown

Reply:
Good idea, I've gone ahead and changed all as well.